### PR TITLE
[Feature Request] Make changeable query execution target

### PR DESCRIPTION
**Pull request by joker1007**
I want to execute query on different database from `ActiveRecord::Base.connection`.
I propose `Adhoq.config.database_connection`.
Even if database for analyzing is different from production database, We will be able to use `adhoq`

for example

```ruby
Adhoq.configuration do |c|
  c.database_connection = -> { OtherDatabaseModel.connection }
end
```

### CODE REVIEW

File: gemfiles/Gemfile-rails-3.2.x [hunk 1]
```diff
@@ -7,6 +7,8 @@ gem 'rails', '~> 3.2.0'
 gem 'font-awesome-sass', '4.2.0'
 gem 'strong_parameters'
 
+gem 'test-unit'
```
Comment (moro) [inline gemfiles/Gemfile-rails-3.2.x]: Oh... Really need it on Rails 3.2? :scream:

Comment (joker1007) [inline gemfiles/Gemfile-rails-3.2.x]: I need it on ruby-2.2 & rails-3.2.
But I think that better way may be exist.

Comment (moro) [inline gemfiles/Gemfile-rails-3.2.x]: I see. Let it be at this time.

**Comment by moro** [comment]
Thanks a lot! 

I'm sorrty to often ask, can you resolve conflict? Then I'll merge this.

**Comment by joker1007** [comment]
:ok:  I rebased now.
Please check it.

**Comment by moro** [comment]
Thaks :meat_on_bone: :sheep:

**Pull request merged**